index.ts

```diff
@@ -214,18 +210,25 @@ async function exportSingleEvent(
         customerPayload.created_at = Date.parse(customerPayload.created_at) / 1000
     }
 
-    let id = event.distinct_id
+    let id = event.person.uuid
 
     if (customer.email) {
         customerPayload.email = customer.email
         if (identifyByEmail) {
             id = customer.email
         }
     }
+
     // Create or update customer
     // See https://www.customer.io/docs/api/#operation/identify
     await callCustomerIoApi('PUT', host, `/api/v1/customers/${id}`, authorizationHeader, customerPayload)
 
+    if (['$identify', '$merge_dangerously'].includes(event.event) && !event.$merge_blocked) {
```

Comment:
also `$create_alias` ; `$merge_blocked` will be within event.properties <- if properties don't exist, then we should assume merge wasn't blocked